FlorenceBML/BMLacq680/BMLacq680.xml

```diff
@@ -35,6 +36,578 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Marrassini ms. 15</idno>
```

Comment:
I have inserted empty pages for missing images, please change `n="309"` in `facs` above

(`<idno facs="Laurenziana/BML_015/BML_015-" n="309">BML Acq. e doni 680</idno>`)

Reply:
Ok. I changed 307 for 309 in idno.